website/docs/r/is_instance.html.markdown

```diff
@@ -295,7 +295,6 @@ Review the argument references that you can specify for your resource.
       - Use this only if you have **IP spoofing operator** access.
 
   - `name` - (Optional, String) The name of the network interface.
-  - `port_speed` - (Deprecated, Integer) Speed of the network interface.
```

Comment:
With this removal, lets make sure we remove the argument support from datasource https://github.com/ibm-vpc/terraform-provider-ibm/blob/7d21dbe45b7beb4b962702a4c63eee5bde9c1df0/ibm/resource_ibm_is_instance.go#L317